src/library/scala/math/Numeric.scala

```diff
@@ -194,13 +184,12 @@ object Numeric {
     def rem(x: BigDecimal, y: BigDecimal): BigDecimal = x remainder y
   }
 
-  // For Double and BigDecimal we offer implicit Fractional objects, but also one
+  // For BigDecimal we offer implicit Fractional objects, but also one
```

Comment:
Should probably be singular "an implicit Fractional object" (admittedly, it wasn't grammatically correct in the first place)